### DISCUSSION TITLE
Tileset3D Frustum Culling for Babylon

### DISCUSSION THREAD

**Discussion by swarmt** [visgl/loaders.gl#2197] [Q&A]
Hi all,

I'm looking to use the Tileset3D loader with Babylon.js.

From what I have researched it appears the best way to do this is to use Babylonjs' GLTFFileLoader to load the parsed gltfArrayBuffer which makes sense.

Can someone suggest the easiest method to update the active tiles the culling volume from BabylonJS's camera?
Particularly how to handle the transforms?

Thanks

**Comment by kai-dorschner-twinsity** [comment]
I would also be interested in this topic
